javalib/src/main/scala/java/util/concurrent/TimeUnit.scala

```diff
@@ -0,0 +1,129 @@
+package java.util.concurrent
+
+// copy&paste from Scala.js
```

Comment:
IMO for *attribution* nothing is really required. It's all Copyright EPFL anyway (through the CLA), so it doesn't matter.

I think the comment is relevant to readers of the source code that, if they find something to change there, there's a high probability it would also require a change in Scala.js.